plugins/packages/polygon/snapshot.go

```diff
@@ -167,9 +167,14 @@ func (service *PolygonBorService) GetSnapshot(ctx context.Context, blockNrOrHash
 		log.Error("Error getting recents get_snapshot()", "err", err.Error())
 	}
 
-	mod := blockNumber % 64
+	var sprint uint64
```

Comment:
Don't you want to set sprint to `64` or `16` rather than blockNumber % 64 or blockNumber % 16?

Because:

```
switch sprint {
case sprint -1:
  // This will never match
}
```